app/controllers/products_controller.rb

```diff
@@ -4,6 +4,10 @@ class ProductsController < ApplicationController
   # GET /products or /products.json
   def index
     @products = Product.all.order(:title)
+    respond_to do |format|
+      format.html
+      format.json
```

Comment:
read about the method `to_json` and see if it can be called on products

Reply:
Yes it can be called on products
Used different approach, json is being called on products